changelog/5657.bugfix.md

```diff
@@ -0,0 +1,17 @@
+Add List handling in the `send_custom_json` method on `channels/facebook.py`.
+When the custom json message has, for instance, 2 keys with the same name
+under `blocks` -- see the example below, instead of receiving a Dict, the
+`send_custom_json` method receives a List.
```

Comment:
Does this imply that that when the key `blocks` has a `list` as a value, the whole json is wrapped in a list? If that's so, the example should actually look like this: `[ { "blocks": [...] } ]`